Dockerfile

```diff
@@ -1,37 +1,33 @@
-FROM registry.opensource.zalan.do/stups/python:3.5-cd28
+FROM registry.opensource.zalan.do/stups/openjdk:1.8.0-222-22
 MAINTAINER Team Aruha, team-aruha@zalando.de
 
-ENV KAFKA_VERSION="1.1.1" SCALA_VERSION="2.12" JOLOKIA_VERSION="1.3.3"
+ENV KAFKA_VERSION="2.3.0" SCALA_VERSION="2.12" JOLOKIA_VERSION="1.3.3"
+ENV KAFKA_LOGS_DIR="/data/logs"
 ENV KAFKA_DIR="/opt/kafka"
+ENV HEALTH_PORT=8080
+ENV KAFKA_SETTINGS="${KAFKA_DIR}/config/server.properties"
+ENV BUKU_FEATURES="restart_on_exhibitor,rebalance_on_start,graceful_terminate,use_ip_address"
+ENV KAFKA_OPTS="-server -Dlog4j.configuration=file:${KAFKA_DIR}/config/log4j.properties -Dkafka.logs.dir=${KAFKA_LOGS_DIR} -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=32M -javaagent:/opt/jolokia-jvm-agent.jar=host=0.0.0.0"
+ENV KAFKA_JMX_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false"
 
-RUN apt-get update && apt-get install wget openjdk-8-jre -y --force-yes && apt-get clean
 ADD docker/download_kafka.sh /tmp/download_kafka.sh
 
-RUN sh /tmp/download_kafka.sh ${SCALA_VERSION} ${KAFKA_VERSION} ${KAFKA_DIR}
+RUN sh /tmp/download_kafka.sh ${SCALA_VERSION} ${KAFKA_VERSION} ${KAFKA_DIR} ${JOLOKIA_VERSION} && apt-get update && apt-get install -y python3-pip 
 
 ADD docker/server.properties ${KAFKA_DIR}/config/
-
-RUN wget -O /tmp/jolokia-jvm-agent.jar http://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/$JOLOKIA_VERSION/jolokia-jvm-$JOLOKIA_VERSION-agent.jar
-
-ENV KAFKA_OPTS="-server -Dlog4j.configuration=file:${KAFKA_DIR}/config/log4j.properties -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=32M -javaagent:/tmp/jolokia-jvm-agent.jar=host=0.0.0.0"
-ENV KAFKA_JMX_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false"
-
-RUN mkdir -p $KAFKA_DIR/logs/
-
-ENV KAFKA_SETTINGS="${KAFKA_DIR}/config/server.properties"
 ADD docker/server.properties ${KAFKA_SETTINGS}
 ADD docker/log4j.properties ${KAFKA_DIR}/config/
 
 ENV SRC_PATH="/bubuku"
 
-RUN mkdir -p "${SRC_PATH}"
-WORKDIR "${SRC_PATH}"
 ADD ./bubuku "${SRC_PATH}/bubuku"
 ADD ./requirements.txt "${SRC_PATH}/"
 ADD ./setup.py "${SRC_PATH}/"
-RUN pip3 install --no-cache-dir -r requirements.txt
+RUN cd "${SRC_PATH}" && pip3 install --no-cache-dir -r "requirements.txt" && python3 setup.py develop
 
 EXPOSE 9092 8080 8778 8888
```

Comment:
why do we need `8888` ?

Reply:
This is default bubuku health port

Reply:
`ENV HEALTH_PORT=8080`
https://github.com/zalando-nakadi/bubuku/pull/144/files#diff-3254677a7917c6c01f55212f86c57fbfR7
what about that then ?

Reply:
@adyach Removed it from config